### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by weinventyou**
Added info-overlay text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,9 +25,26 @@
 
 <!--<div id="overlayGIF"></div> overlayGIF details in main.css-->
     <div id="info-overlay">
+    
         <div id="text">
-            <B>THIRTY SECOND STORIES</B><BR><BR>
+            <div id="title">THIRTY SECOND STORIES</div><p>
```
Comment (kyledemeule) [inline index.html]: Does this id get used anywhere?

File: index.html [hunk 2]
```diff
@@ -25,9 +25,26 @@
 
 <!--<div id="overlayGIF"></div> overlayGIF details in main.css-->
     <div id="info-overlay">
+    
         <div id="text">
```
Comment (kyledemeule) [inline index.html]: Also I just noticed this, not a great id name :)
Does this get used for anything?

Comment (weinventyou) [inline index.html]: Yeah, I think that's a Lexy addition to the css from last year. It only gets used for the overlay text.

File: index.html (hunk 1 above)
Comment (weinventyou) [inline index.html]: I added that (following Lexy's model, probably foolishly!) and it's just for the Title at the top of the info-overlay, which is bigger than the rest of the text.

**Pull request merged**